library/constant_time.c

```diff
@@ -78,11 +78,11 @@ static inline uint32_t mbedtls_get_unaligned_volatile_uint32(volatile const unsi
      */
     uint32_t r;
 #if defined(MBEDTLS_CT_ARM_ASM)
-    asm volatile ("ldr %0, [%1]" : "=r" (r) : "r" (p) :);
+    __asm volatile ("ldr %0, [%1]" : "=r" (r) : "r" (p) :);
 #elif defined(MBEDTLS_CT_AARCH64_ASM)
     asm volatile ("ldr %w0, [%1]" : "=r" (r) : MBEDTLS_ASM_AARCH64_PTR_CONSTRAINT(p) :);
 #else
-#error No assembly defined for mbedtls_get_unaligned_volatile_uint32
+#error "No assembly defined for mbedtls_get_unaligned_volatile_uint32"
```

Comment:
Good catch, but there are several of these in the code base, so can you fix all of them, not just this one please?

Reply:
Yes, that's fine

Reply:
Will you do that?

Reply:
Thanks!

Reply:
Added a separate commit to this PR, just shout in case you'd want a separate PR.

Reply:
It's fine to have them all in this PR, they came up while fixing IAR compilation

Reply:
Err, I also see

```
library/common.h:#error Unrecognised pointer size for aarch64
library/psa_crypto_storage.h:#error PSA_CRYPTO_MAX_STORAGE_SIZE > 0xffff0000
```

Reply:
ah, I grepped C files only. Will fix those in a bit.